package.json

```diff
@@ -1,18 +1,41 @@
 {
   "name": "Snipper",
   "version": "1.0.0",
-  "description": "",
+  "description": "A cross platform desktop app that lets you save your frequently used code snippets.",
   "main": "./main.js",
   "scripts": {
+  	"postinstall": "install-app-deps",
+  	"pack": "build --dir",
+    "dist-mac": "build --mac",
     "test": "echo \"Error: no test specified\" && exit 1",
     "start": "electron ."
   },
+  "build": {
+    "appId": "coding-blocks.Snipper",
```

Comment:
can an App Icon be also added (use a dummy square icon of CB for now, we will design a icon separately later)